### ISSUE TITLE
All methods and functionality should follow the javadoc comment format

### ISSUE THREAD

**Issue by marcusalstrom** [mebn/DD2480#85]
https://www.oracle.com/se/technical-resources/articles/java/javadoc-tool.html

**Comment by mebn** [comment]
I don't think this is neccesary for assignment 1, unless i missed someithng. However, we have to do it for assignment 2 
<img width="831" alt="bild" src="https://github.com/mebn/DD2480/assets/39590503/80721c70-1f29-4841-9468-36a2549f98be">
